test/package.spec.ts

```diff
@@ -2,20 +2,50 @@
  * Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
  */
 
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import Package from '../src/commands/package';
+import { getCleanProjectDir, getTestOutputDir } from './test-utils';
+
+const testOutputDir = getTestOutputDir(__filename);
+const nestAppDir = path.resolve('test', 'nest');
+
 describe('Package', () => {
-  it('should copy files correctly without parameters', () => {
-    expect(true).toBeTruthy();
+  afterAll(() => {
+    // fs.removeSync(testOutputDir);
   });
 
-  it('should copy files correctly with custom globs', () => {
-    expect(true).toBeTruthy();
+  it('should copy files correctly without parameters', async () => {
+    const projectDir = getCleanProjectDir(testOutputDir, 'no-params');
+    fs.copySync(nestAppDir, projectDir, { recursive: true });
+    await Package.run([projectDir, '--skipInstall']);
+
+    expect(fs.readdirSync(path.resolve(projectDir, 'deployment'))).toIncludeAllMembers(['package.json', 'package-lock.json']);
   });
 
-  it('should overwrite output folder', () => {
-    expect(true).toBe(true);
+  it('should copy files correctly with custom globs', async () => {
+    const projectDir = getCleanProjectDir(testOutputDir, 'globs');
+    fs.copySync(nestAppDir, projectDir, { recursive: true });
+    await Package.run([projectDir, '--include=*.json', '--exclude=package*,tsconfig*', '--skipInstall']);
+
+    expect(fs.readdirSync(path.resolve(projectDir, 'deployment'))).toIncludeAllMembers(['nest-cli.json', 'tslint.json']);
```

Comment:
Similar to the comment above, if you set the `--exclude` flag, I would've expected the test to also assert that the excluded files are not present in the target directory.